http/rest-client-reactive/src/main/java/io/quarkus/ts/http/restclient/reactive/BookClient.java

```diff
@@ -13,14 +15,15 @@
 
 import org.eclipse.microprofile.rest.client.annotation.RegisterClientHeaders;
 import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+import org.jboss.resteasy.reactive.RestForm;
 import org.jboss.resteasy.reactive.RestQuery;
 
 import io.quarkus.ts.http.restclient.reactive.json.Book;
 import io.quarkus.ts.http.restclient.reactive.json.BookIdWrapper;
 import io.smallrye.mutiny.Multi;
 import io.smallrye.mutiny.Uni;
 
-@RegisterRestClient
+@RegisterRestClient(configKey = "BookClient")
```

Comment:
Is this needed here. I try it without it and it works. Setting config key should be only needed if you using some key in property file to not use full class path.

We setting in property file url like this  `quarkus.rest-client."io.quarkus.ts.http.restclient.reactive.BookClient".url`  so it's not needed to set configKey.

If it's set here for reason I think the property file should be modified.

Reply:
You're right, now here is not needed. I was playing and testing around with it and the properties in dev mode because upstream issue #43784 has this kind of configuration.